Source/WebCore/html/PDFDocument.cpp

```diff
@@ -45,6 +45,7 @@
 #include "RawDataDocumentParser.h"
 #include "ScriptController.h"
 #include "Settings.h"
+#include <JavaScriptCore/ObjectConstructor.h>
```

Comment:
Was this change intentional?

Reply:
Yes.  This is needed to fix a build failure caused by removing a header included elsewhere.